ktor-server/ktor-server-plugins/ktor-server-openapi/jvm/src/io/ktor/server/plugins/openapi/OpenAPIConfig.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.plugins.openapi
+
+import io.swagger.codegen.v3.*
+import io.swagger.codegen.v3.generators.html.*
+import io.swagger.parser.*
+import io.swagger.v3.parser.core.models.*
+
+/**
+ * Configuration for OpenAPI endpoint.
+ */
+public class OpenAPIConfig {
```

Comment:
minor: function to configure called `openApi` but config is `OpenAPI*` - may use single way of exposing abbreviations?

Reply:
Thanks, fixed to `API`